http-client/src/lib.rs

```diff
@@ -1,3 +1,12 @@
+#[cfg(not(any(feature = "tokio1", feature = "tokio02")))]
+compile_error!("Either feature \"tokio1\" or \"tokio02\" must be enabled for this crate.");
+
+#[cfg(feature = "tokio1")]
+extern crate hyper14 as hyper;
```

Comment:
Is this here so we can do conditional crate renaming?

Reply:
indeed, do have any better suggestion?

Reply:
so something like `use X as Y` won't work with a `#[cfg()]` atop?

Reply:
> so something like `use X as Y` won't work with a `#[cfg()]` atop?

Nope, I just tried. Compiler gets super angry. Weird.

Reply:
yeah, we don't have any crate named `hyper` in the dependency tree anymore.

I guess another solution is that move the `#[cfg()]` in to the struct `HttpTransportClient`